src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -490,12 +490,12 @@ private void ValidateHash(string expectedValue, string hashItem, string algorith
         /// <summary>
         /// Validates the 'code' according to http://openid.net/specs/openid-connect-core-1_0.html
         /// </summary>
-        /// <param name="validationContext">a <see cref="OpenIdConnectProtocolValidationContext"/> that contains the protocol message to validate.</param>
-        /// <exception cref="ArgumentNullException">if 'idToken' is null.</exception>
-        /// <exception cref="ArgumentNullException">if 'validationContext' is null.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidCHashException">if the validationContext contains a 'code' and there is no 'c_hash' claim in the 'id_token'.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidCHashException">if the validationContext contains a 'code' and the 'c_hash' claim is not a string in the 'id_token'.</exception> 
-        /// <exception cref="OpenIdConnectProtocolInvalidCHashException">if the 'c_hash' claim in the 'idToken' does not correspond to the 'code' in the <see cref="OpenIdConnectMessage"/> response.</exception> 
+        /// <param name="validationContext">A <see cref="OpenIdConnectProtocolValidationContext"/> that contains the protocol message to validate.</param>
+        /// <exception cref="ArgumentNullException">If 'idToken' is null.</exception>
+        /// <exception cref="ArgumentNullException">If 'validationContext' is null.</exception>
```

Comment:
change order: 'If validationContext ...' before 'if idToken...'
change text: 'If 'idToken'  is null' to 'If validationContext.ValidatedIdToken is null'